app/models/manageiq/providers/azure/manager_mixin.rb

```diff
@@ -126,7 +128,7 @@ def params_for_create
     #   }
     # }
     def verify_credentials(args)
-      region           = args["region"]
+      region           = args["provider_region"]
```

Comment:
was this a bug?

Reply:
Yes, I have a separate PR to fix just this, the second commit in this branch builds on that fix

Reply:
https://github.com/ManageIQ/manageiq-providers-azure/pull/496